client/app/app.js

```diff
@@ -5,33 +5,50 @@ var initMap = function(){
     zoom: 14
   });
 
-  // var geocoder = new google.mapsGeocoder();
+  var geocoder = new google.maps.Geocoder();
+  //test data array
```

Comment:
Hi Alice, everything looks great. Could you please add more pseudocode for documentation purposes? Explanations of what each of these google maps api functions are doing and how they're connected to each other might help the legacy group inherit our project.